playbooks/files/get_url/get_url.py

```diff
@@ -12,13 +12,14 @@ def main():
         args = yaml.safe_load(file)
     client = statsd.StatsClient(args["host"], args["port"])
     while True:
+        metric_name = args["metric_name"]
         for host in args["nodes"]:
             metric_name = f'{args["statsd_prefix"]}.{args["metric_name"]}.{host["name"]}'
             try:
                 res = requests.get(host["ip"], headers={'Connection': 'close'}, timeout=5)
                 client.timing(f'{metric_name}', res.elapsed.total_seconds() * 1000)
             except Exception as ex:
-                client.incr(f'counters.{metric_name}.failed')
+                client.incr(f'counters.{metric_name}.failed', 10)
```

Comment:
why is that?

Reply:
i dont know but default value=1, appears in graphite as 0.1